test/e2e/authz-templated-query-rewrite-static/clusterRole.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
revert all of the file moves, like [test/e2e/static-auth/serviceAccount.yaml → test/e2e/authz-static/serviceAccount.yaml](https://github.com/brancz/kube-rbac-proxy/pull/377/commits/b8d2d7efc658877f0f1178a062a96f21e0c57fc9#diff-421e293188e06df7e377bb2fb4ce47028e002e841aa725b4b3b89c3c3d224dcd)
This is really hard to review otherwise and the benefit of the move is minimal